### PR TITLE
Temp#32190

### PR DESCRIPTION
**Pull request by NikitaEmberi**
Pull Request for Issue #32190 

### Summary of Changes
Changes as suggested in: https://github.com/joomla/joomla-cms/issues/32190#issuecomment-801452820 

### Actual result BEFORE applying this Pull Request
image

### Expected result AFTER applying this Pull Request

image

No changes were made to _icon.scss file but I don't why it keeps showing up in "files changed" section.

### CODE REVIEW

**Comment by Quy** [comment]
Please do a global find on and change accordingly:

`<td class="text-center tbody-icon">`

File: administrator/templates/atum/scss/blocks/_icons.scss [hunk 1]
```diff
@@ -63,7 +63,7 @@
     cursor: not-allowed;
     opacity: 1;
   }
-
+
```
Comment (richard67) [inline administrator/templates/atum/scss/blocks/_icons.scss]:
```suggestion
```
Revert this change to fix unnecessary code style error reported by drone CI.

And please give your PR a meaningful title.

File: administrator/templates/atum/scss/blocks/_icons.scss [hunk 2]
```diff
@@ -63,7 +63,6 @@
     cursor: not-allowed;
     opacity: 1;
   }
-
```
Comment (richard67) [inline administrator/templates/atum/scss/blocks/_icons.scss:66]: I just see it needs to add back the empty line, it was removed due to my previous review which should just remove the spaces in this empty line. My mistake, sorry.

**Comment by jwaisner** [comment]
Test is successful with your PR.

image

**Comment by jwaisner** [comment]
I have tested this item :white_check_mark: successfully on 07e6192388d40dfbe8df134d8a1f8df13cb62d21<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/32762">issues.joomla.org/tracker/joomla-cms/32762</a>.</sub>

**Comment by rdeutz** [comment]
continue working on #32789
closeing as duplicate

**Pull request closed without merging**